src/Adapter/Product/StockConfiguration.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Adapter\Product;
+
+use PrestaShop\PrestaShop\Adapter\Configuration;
+use PrestaShop\PrestaShop\Core\Configuration\DataConfigurationInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class StockConfiguration implements DataConfigurationInterface
+{
+    /**
+     * @var Configuration
+     */
+    private $configuration;
+
+    public function __construct(Configuration $configuration)
```

Comment:
would you mind to rely on ConfigurationInterface instead? This one =>https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/ConfigurationInterface.php

Sorry if it's done with Configuration elsewhere, this needs to be updated/improved (my bad).

Reply:
Yes! :+1:  Fixed.